GG_testing_channels.md

```diff
@@ -0,0 +1,45 @@
+As developers we typically value tests since they help to 'future-proof' our applications by
+minimizing regression and provide updated documentation. Phoenix recognizes this and helps
+make it easier to write tests by providing conveniences for testing its different parts,
+including Channels.
+
+In the Channels guide it was mentioned that a "Channel" is a layered system with different
```

Comment:
What would you think about changing the first bit of this sentence to read something like this:

In the Channels Guide, we saw that . . .